lib/matplotlib/tests/test_transforms.py

```diff
@@ -1104,3 +1105,26 @@ def test_interval_contains_open():
     assert not mtransforms.interval_contains_open((0, 1), -1)
     assert not mtransforms.interval_contains_open((0, 1), 2)
     assert mtransforms.interval_contains_open((1, 0), 0.5)
+
+
+def test_initialization():
```

Comment:
```suggestion
def test_scaledrotation_initialization():
```

Just a bit more specific, perhaps also add something similar to the other test below